CONTRIBUTING.md

```diff
@@ -211,6 +211,23 @@ sudo apt-get update
 sudo apt-get install python2.7 python2.7-dev
 ```
 
+In Ubuntu, sometimes even after installing Node.js, the command 
+
+```
+node -v
```

Comment:
I think here you can just use an inline code comment, like so:

"...installing Node.js, the command `node -v` does not show...`. 

To do that, you just need 1 set of backticks \`node -v\`